### PR TITLE
In the News page done

### PR DESCRIPTION
**Pull request by Abibibi**

### CODE REVIEW

File: src/pages/MediaNews/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import data from './data'
+
+import DefaultLayout from '../../layout/Default'
+
+const MediaNews = () => {
+    const { t } = useTranslation('media')
+
+    return (
+        <DefaultLayout title={`${t('page-title')} - Instant Visio`}>
+            <MediaNewsTitle>On parle de nous</MediaNewsTitle>
```
Comment (ntchou) [inline src/pages/MediaNews/index.js:13]: missing translation

File: public/locales/gr/media.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "page-title": "On parle de nous",
```
Comment (HugoGresse) [inline public/locales/gr/media.json:2]: Maybe worth using english for all non french / non translated lang?

File: src/pages/MediaNews/index.js (hunk 1 above)
Comment (Abibibi) [reply src/pages/MediaNews/index.js]: Done

File: public/locales/gr/media.json (hunk 2 above)
Comment (Abibibi) [reply public/locales/gr/media.json]: Yes! Done

**Pull request merged**